src/core/Akka/Dispatch/AbstractDispatcher.cs

```diff
@@ -241,7 +441,62 @@ public virtual void SystemDispatch(ActorCell cell, Envelope envelope)
         /// <param name="cell">The ActorCell belonging to the actor who's attaching to this dispatcher.</param>
         public virtual void Attach(ActorCell cell)
         {
-            
+            Register(cell);
+            RegisterForExecution(cell.Mailbox, false, true);
```

Comment:
Turns on the mailbox for the very first time when an actorcell attaches to this dispatcher.